src/domain/landingPage/landingPageMeta/LandingPageMeta.tsx

```diff
@@ -1,39 +1,57 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router';
 
 import { LandingPageFieldsFragment } from '../../../generated/graphql';
 import useLocale from '../../../hooks/useLocale';
+import { ROUTES } from '../../app/routes/constants';
 import { getBannerFields } from '../../banner/bannerUtils';
 import { getLandingPageFields } from '../utils';
+
 interface Props {
   landingPage: LandingPageFieldsFragment;
 }
 
 const LandingPageMeta: React.FC<Props> = ({ landingPage }) => {
   const locale = useLocale();
+  const { pathname } = useLocation();
+  const { t } = useTranslation();
 
   const { keywords, pageTitle, metaInformation } = getLandingPageFields(
     landingPage,
     locale
   );
+
   const { someImage: topBannerSoMeImage } = getBannerFields(
     locale,
     landingPage.topBanner
   );
+
   const { someImage: bottomBannerSoMeImage } = getBannerFields(
     locale,
     landingPage.bottomBanner
   );
+
   const openGraphProperties = {
     description: metaInformation,
     image: [topBannerSoMeImage, bottomBannerSoMeImage],
     title: pageTitle,
   };
 
+  const getDescription = () => {
+    const route = [
+      ROUTES.COURSES,
+      ROUTES.EVENTS,
+      ROUTES.COLLECTIONS,
+    ].find((route) => pathname.startsWith(`/${locale}${route}`));
+    return t(`meta.${route?.substring(1) || 'default'}.description`);
```

Comment:
As far as I understand this is not what is wanted. This `LandingPageMeta` is part of the `LandingPage` and `LandingPagePreview`, so this should perhaps contain only default description. There is also `CollectionPageMeta` for collections page and `EventPageMeta` for courses and events pages. Also I found `StaticPageMeta` for accessibility and about page. Not sure what to do with them. Perhaps default text to them? I asked about it here: https://helsinkicity.slack.com/archives/C013682SYSD/p1616757237019800

Reply:
they work differently, those you mentioned are coming from Wagtail. To make proper implementation (we discussed it with Khin and Niko) we need to make changes to the wagtail arch, the description was not supported for multiple Landing pages, but its supported for all other types. But its a bigger change, Mari suggested that we keep it easy, as the cms will be changed soon anyway.